components/centraldashboard/public/components/activities-list_test.js

```diff
@@ -0,0 +1,102 @@
+import {flush} from '@polymer/polymer/lib/utils/flush.js';
+import '@polymer/test-fixture/test-fixture';
+
+import './activities-list';
+
+const ONE_DAY = 24 * 60 * 60 * 1000;
+const FIXTURE_ID = 'activities-list-fixture';
+const ACTIVITIES_LIST_ID = 'test-activities-list';
+const TEMPLATE = `
+<test-fixture id="${FIXTURE_ID}">
+  <template>
+    <activities-list id="${ACTIVITIES_LIST_ID}"></activities-list>
+  </template>
+</test-fixture>
+`;
+
+describe('Activities List', () => {
+    let activitiesList;
+
+    beforeAll(() => {
+        const div = document.createElement('div');
```

Comment:
We could re-use `lit-html` here!

Reply:
I played around with this a bit and I think we probably could eliminate this pattern going forward. It will probably make sense to add some additional helper functions for setup and tear down work in unit tests.